sdk/ai/azure-ai-assistants/tests/model_assistants_test_base.py

```diff
@@ -0,0 +1,493 @@
+    #     )
+
+    # # Methods to create the different sync and async clients directly
+    # def _create_async_assistant_client(self, *, bad_key: bool = False, **kwargs) -> async_sdk.AssistantsClient:
+    #     endpoint, credential = self._load_chat_credentials(bad_key=bad_key, **kwargs)
+    #     return async_sdk.AssistantsClient(
+    #         endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED, **kwargs
+    #     )
+
+    # def _create_assistant_client(
+    #     self, *, key_auth: bool = True, bad_key: bool = False, **kwargs
+    # ) -> sdk.AssistantsClient:
+    #     endpoint, credential, credential_scopes, headers, api_version = self._load_aoai_chat_credentials(
+    #         key_auth=key_auth, bad_key=bad_key, **kwargs
+    #     )
+    #     return sdk.AssistantsClient(
+    #         endpoint=endpoint,
+    #         credential=credential,
+    #         credential_scopes=credential_scopes,
+    #         headers=headers,
+    #         api_version=api_version,
+    #         logging_enable=LOGGING_ENABLED,
+    #     )
+
+    def _create_async_aoai_chat_client(
+        self, *, key_auth: bool = True, bad_key: bool = False, **kwargs
+    ) -> async_sdk.AssistantsClient:
+        endpoint, credential, credential_scopes, headers, api_version = self._load_aoai_chat_credentials(
+            key_auth=True, bad_key=bad_key, **kwargs
+        )
+        return async_sdk.AssistantsClient(
+            endpoint=endpoint,
+            credential=credential,
+            credential_scopes=credential_scopes,
+            headers=headers,
+            api_version=api_version,
+            logging_enable=LOGGING_ENABLED,
+        )
+
+    # def _create_async_embeddings_client(self, *, bad_key: bool = False, **kwargs) -> async_sdk.EmbeddingsClient:
+    #     endpoint, credential = self._load_embeddings_credentials(bad_key=bad_key, **kwargs)
+    #     return async_sdk.EmbeddingsClient(
+    #         endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED, **kwargs
+    #     )
+
+    # def _create_embeddings_client(self, *, bad_key: bool = False, **kwargs) -> sdk.EmbeddingsClient:
+    #     endpoint, credential = self._load_embeddings_credentials(bad_key=bad_key, **kwargs)
+    #     return sdk.EmbeddingsClient(endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED, **kwargs)
+
+    # def _create_embeddings_client_with_chat_completions_credentials(self, **kwargs) -> sdk.EmbeddingsClient:
+    #     endpoint = kwargs.pop("azure_ai_chat_endpoint")
+    #     key = kwargs.pop("azure_ai_chat_key")
+    #     credential = AzureKeyCredential(key)
+    #     return sdk.EmbeddingsClient(endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED)
+
+    def request_callback(self, pipeline_request) -> None:
+        self.pipeline_request = pipeline_request
+
+    def _validate_embeddings_json_request_payload(self) -> None:

```

Comment:
There is no embeddings concept with assistants, at least currently. We can remove all embeddings related things from assistants tests.
There is FileSearch tool for assistants that can be used to create RAG scenario and where the vectorized embeddings are created inside assistant service (not visible to user)